internal/datastore/mysql/datastore.go

```diff
@@ -214,6 +214,11 @@ func newMySQLDatastore(ctx context.Context, uri string, replicaIndex int, option
 	// ID value makes this idempotent (i.e. safe to execute concurrently).
 	createBaseTxn := fmt.Sprintf("INSERT IGNORE INTO %s (id, timestamp) VALUES (1, FROM_UNIXTIME(1))", driver.RelationTupleTransaction())
 
+	headMigration, err := migrations.Manager.HeadRevision()
+	if err != nil {
+		return nil, fmt.Errorf("invalid head migration found for mysql: %w", err)
+	}
+
```

Comment:
Was this a reorganization or was it required by the refactor?

Reply:
Yeah, previously this was called indirectly by

```
compatible, err := migrations.Manager.IsHeadCompatible(currentMigrationRevision)
```